packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/sdk.md

```diff
@@ -1,16 +1,20 @@
 ### 1: Initiate password recovery
 
-To begin the password recovery flow, the user clicks **Forgot Password** on the sign-in page. Then, they select **Email** as the authenticator they want to use for password recovery on the subsequent page and click **Submit**. Okta then sends an email to the email address that matches the Forgot Password template that was altered earlier.
+The user starts the password recovery flow with the following steps:
+
+1. Clicks the **Forgot Password?** link on the sign-in page.
+2. Enters their **Email or Username** in the dialog and then clicks **Next**.
+3. Chooses **Email** as the authenticator they want to use for password recovery and clicks **Submit**.
+
+Okta then sends an email to the email address that matches the Forgot Password template that was altered earlier.
```

Comment:
This is the key bit of information that the developer cares about... after the user requests a reset, the Okta org sends something. Perhaps: 
```suggestion
After the developer requests a password reset, Okta sends an email that uses the Forgot Password template (that was altered earlier).
```

Ideally, this text belongs in the section on updating the Forgot Password template. Whether it goes before or after modifying the template, I'm not sure.